moped-database/views/project_list_view.sql

```diff
@@ -159,9 +152,7 @@ AS WITH project_person_list_lookup AS (
      LEFT JOIN moped_entity mel ON mel.entity_id = mp.project_lead_id
      LEFT JOIN moped_proj_partners mpp2 ON mp.project_id = mpp2.project_id AND mpp2.is_deleted = false
      LEFT JOIN moped_entity me2 ON mpp2.entity_id = me2.entity_id
-     left join moped_proj_work_activity mpwa on mpwa.project_id = mp.project_id
-     LEFT JOIN LATERAL jsonb_array_elements(mpwa.task_orders) task_order_filter(value) ON true
```

Comment:
this the only problem—the lateral join needed to be in the subquery. in the rest of the diff i have consolidated all the work activity fields into the one CTE.